docs/scripts/degen.rst

```diff
@@ -0,0 +1,67 @@
+degen
+=====
+Find genes where a sequence has degenerate bases.
+
+How-to
+=====
+Usage:
+    degen.py <fasta> (--gb-id <accession_id> | --gb-file <gbfile> | --tab-file <tabfile>)
+
+Options:
+    --gb-id=<accession_id>   Accession id for reference
+    --gb-file=<gbfile>       Local Genbank file for reference
+    --tab-file=<tabfile>     TSV/CSV file for reference with fields name,start,end
+
+Example:
+=======
+    .. code-block:: bash
+
+        degen sequence.fasta --gb-id   12398.91
+        degen sequence.fasta --gb-file  tests/testinput/sequence.gb
+        degen sequence.fasta --tab-file tests/testinput/degen.tab
+        degen sequence.fasta --tab-file tests/testinput/degen.csv
+
+
+Output:
```

Comment:
I think you need to wrap this in a code block. The sphinx html is all on one line